bundles/catalogue/metadatasearch/view/advanced-search/AdvancedSearchDropdown.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { AdvancedSearchInputLabel, AdvancedSearchRowContainer, AdvancedSearchSelect } from './AdvancedSearchStyledComponents';
+import { Option } from 'oskari-ui';
+import PropTypes from 'prop-types';
+export const AdvancedSearchDropdown = (props) => {
+    const { title, options, onChange, selected } = props;
+    const hasOptions = options && options?.values?.length && options.values.length > 0;
+    return <AdvancedSearchRowContainer>
```

Comment:
I think we can just return null and not render the label nor dropdown if we don't have any values to show for the user.

Reply:
true